CHANGELOG.md

```diff
@@ -6,6 +6,13 @@
 
 * [#8912](https://github.com/rubocop-hq/rubocop/pull/8912): Fix `Layout/ElseAlignment` for `rescue/else/ensure` inside `do/end` blocks with assignment. ([@miry][])
 * [#8971](https://github.com/rubocop-hq/rubocop/issues/8971): Fix a false alarm for `# rubocop:disable Lint/EmptyBlock` inline comment with `Lint/RedundantCopDisableDirective`. ([@koic][])
+### New features
+
+### Bug fixes
+
+* [#8966](https://github.com/rubocop-hq/rubocop/issues/8966): Fix `Layout/SpaceInsideParens` to enforce no spaces in empty parens for all styles. ([@joshuapinter][])
```

Comment:
Might be better ot add your changelog entry under the new `changelog` folder, to avoid having to deal with changelog conflicts.

Reply:
Oh, this must be _very_ new. I'll move it now...

Reply:
Done! Back over to you.